package.json

```diff
@@ -122,9 +122,9 @@
         "yargs": "^17.5.1"
     },
     "engines": {
-        "pnpm": ">=7"
+        "pnpm": ">=8"
```

Comment:
Seems we need to update this too, not too happy about that but looking at the npm stats it seems like the majority of the user base is already on this version
![image](https://github.com/chartjs/Chart.js/assets/39033624/7636b1c0-c1ab-4f1f-85ff-29f7938aea80)
![image](https://github.com/chartjs/Chart.js/assets/39033624/decd2226-cd18-4815-960e-17a2f6fcae91)
https://www.npmjs.com/package/pnpm?activeTab=versions